src/context-menus/message/warn-piracy.js

```diff
@@ -84,7 +79,17 @@ async function warnPiracyHandler(interaction) {
 	});
 	reportEmbed.setTimestamp(Date.now());
 
+	const transcriptButton = new Discord.ButtonBuilder()
+	.setLabel('Jump!')
+	.setStyle(Discord.ButtonStyle.Link)
+	.setEmoji('📨')
+	.setURL(message.url);
```

Comment:
In this codebase we prefer not to chain component functions like this. Instead each `.set` function should be `component.setWhatever()`